### ISSUE TITLE
Edit hashtag live stream on Twitter

### ISSUE THREAD

**Issue by AWossink** [hatnote/hashtag-search#27]
Would it be possible to set up a twitter account that live streams edits with hashtags?

**Comment by Samwalton9** [comment]
Maybe! Do you mean for a single hashtag, or reporting all hashtags?

**Comment by AWossink** [comment]
I don't know; I have no idea what's possible!

The reason I came up with this is that WMNL will be organising an online writing week to tie in with Leeuwarden/Friesland as European Cultural Capital in 2018. They use the hashtag #LF2018 on twitter, and if we could use that on Wikipedia as well (or something like #Friesland), and that would be live twittered, we could flood twitter with Wikipedia, and raise a lot of awareness of what we're doing, and how we are celebrating this event Wikipedia-style.

So if it could work for a single hasthag, that would already be great, but if it could report all hashtags, that might even be better.

**Comment by slaporte** [comment]
You could build this with a [Wikipedia applet on IFTTT](https://ifttt.com/wikipedia)! The Wikipedia channel has a trigger for "New edit with hashtag", and it can send you a notification in Slack, tweet, etc. Let me know if you build something cool!

**Comment by AWossink** [comment]
I forgot about that! I tried to create something, but when I try to test the applet, I get an error message that there's something wrong with the Wikipedia service. Not sure why this is, and I see no way to check what exactly the error might be.
Currently checking if Wikipedia applets created by others are actually working.

**Comment by AWossink** [comment]
I have set up various Wikipedia applets with different triggers to try out if this is working at all. So far no response at all. Might be because I just do something wrong, but I've kept them as simple as possible so I feel that the problem is not with me. Unfortunately IFTTT doesn't provide any option of reporting problems with applets. I'll wait another day, to see if anything is actually working, otherwise this doesn't seem like a viable route to go.

**Comment by slaporte** [comment]
Oh no! Sorry to hear that isn't working. I can investigate what's up with the IFTTT channel, and let you know if I figure it out.

**Comment by AWossink** [comment]
Yeah, if you have the opportunity to look into it, that would be awesome! There will be a writing week about astronomy starting in 2,5 weeks on NLWP and we set up a hashtag for that, so it would be great if we can use it for that already!

**Comment by AWossink** [comment]
Other Wikipedia applets seem to be working more or less; for example, I can get it to send me an email whenever editor X (in this case, myself) makes an edit. I set up the WP hashtag applet to send me an email whenever any hashtag is used (so I left the hashtag field in the applet empty), but that doesn't work on the same edits. So it seems that there might be something wrong with the hashtag part of the applet.

**Comment by AWossink** [comment]
@slaporte Did you already get the chance to have a look at this?
